packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -103,6 +109,22 @@ export async function runBrokenLinks(
     const adjusted_synthetic_timeout_millis =
       options.total_synthetic_timeout_millis! - 7000;
 
+    const errors: BaseError[] = [];
+
+    // Initialize Storage Client with Error Handling. Set to `null` if
+    // screenshot_condition is 'None'
+    const storageClient = createStorageClientIfStorageSelected(
+      errors,
+      options.screenshot_options!.screenshot_condition
+    );
+
+    // Bucket Validation
+    const bucket: Bucket | null = await getOrCreateStorageBucket(
```

Comment:
Do we do anything with this? Or should it not return anything

Reply:
I'll comment this out for now. I just wanted to show where I was going to use it. Rather than a helper function with no context how it was going to be used.